app.js

```diff
@@ -20,40 +21,14 @@ app.set('view engine', 'ejs');
 app.use(logger('dev'));
 app.use('/static', express.static(path.join(__dirname, 'public')));
 app.use('/pizza', pizzaRoute);
+app.use('/', indexRoute);
```

Comment:
I'm not sure why the index.js file needed to exist. It's usually kosher to just have the home route on app.js (or whatever you called your main file).